frontend/src/Pages/Video/style.css

```diff
@@ -57,6 +58,7 @@
 @media (max-width: 900px) {
   .video {
     padding: 20px;
+    margin-left: 0px;
```

Comment:
I guess for 0 value you don't have to specify the unit

Reply:
Thanks, removed.